packages/console/src/components/Switch/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { forwardRef, HTMLProps, ReactNode } from 'react';
+
+import * as styles from './index.module.scss';
+
+type Props = HTMLProps<HTMLInputElement> & {
+  // eslint-disable-next-line react/require-default-props
+  label?: ReactNode;
+};
+
+const Switch = forwardRef<HTMLInputElement, Props>(({ label, ...rest }, ref) => {
+  const input = (
+    <label className={styles.switch}>
+      <input type="checkbox" {...rest} ref={ref} />
+      <span className={styles.slider} />
+    </label>
+  );
+
+  if (!label) {
+    return input;
+  }
```

Comment:
Any chance we don't have a label for Switch?

Reply:
Currently no, removed.